apps/src/templates/certificates/Certificate.jsx

```diff
@@ -65,7 +65,11 @@ function Certificate(props) {
       course: props.tutorial,
       donor
     };
-    return btoa(reEncodeNonLatin1(JSON.stringify(data)));
+    const encoded = btoa(reEncodeNonLatin1(JSON.stringify(data)));
+    // This method complies with “Base 64 Encoding with URL and Filename
+    // Safe Alphabet” in RFC 4648. The alphabet uses ‘-’ instead of ‘+’ and
+    // ‘_’ instead of ‘/’.
+    return encoded.replace(/\+/g, '-').replace(/\//g, '_').replace(/=+$/g, '');
```

Comment:
This line makes the unit test fail because its expecting a padding = sign at the end of the certificate url.
Should I remove the replacement of the = sign or should I update the test?